edgelet/iotedged/src/unix.rs

```diff
@@ -1,14 +1,30 @@
 // Copyright (c) Microsoft. All rights reserved.
 
-use edgelet_docker::DockerModuleRuntime;
-
 use crate::app;
 use crate::error::Error;
 use crate::signal;
 
+#[cfg(feature = "runtime-docker")]
```

Comment:
The only thing you do with these imports is generate the `ModuleRuntime` alias.

You could save on a lot of typing by just fully-qualifying the imports in the type alias definition.

```rust
#[cfg(feature = "runtime-docker")]
type ModuleRuntime = edgelet_docker::DockerModuleRuntime;

#[cfg(feature = "runtime-kubernetes")]
type ModuleRuntime = edgelet_kube::KubeModuleRuntime<kube_client::ValueToken, ...>;
```

Same for the other file.

Reply:
I think that looks better, overall as well. Done.